### ISSUE TITLE
UI doesn't recognize mouse inputs (Windows 10)

### ISSUE THREAD

**Issue by Requiem365** [fontforge/fontforge#4659]
I am under Windows 10 and Fontforge (20201107) doesn't recognize any mouse inputs.
Wherever I click nothing happens, nor the menu nor any button catch my mouse events. Even the drag bar and the close or minimize buttons don't work.

Oddly the UI reacts almost normaly with my Wacom graphic tablet except the menus. Once they are opened I can not click any item. That makes Fonteforge totally unusable.

That's the only software I have that doesn't work correctly with the mouse.
Usually it's my wacom tablet which causes problems, here it's rather the opposite.
Any idea what's going wrong ?

**Comment by ctrlcctrlv** [comment]
Perhaps some combatibility layer is expecting one pointing device. Try unplugging the Wacom, restarting if necessary after doing so? Just for our information.

**Comment by Requiem365** [comment]
Thanks for the suggestion. Unplugging the Wacom didn't help sadly. And as I use my Wacom 90% of the time rather than the mouse I wouldn't bother if FontForge worked correctly with the tablet but I can not access the menu items using the pen.
If FontForge UI had to fail with a pointing device it should be with the Wacom device, unlikely the mouse that is managed by the system.

**Comment by ctrlcctrlv** [comment]
Hmm. My only guess right now is that the Wacom driver is doing something which is confusing some part of FontForge, maybe GDK

**Comment by Requiem365** [comment]
Yes, most certainly. After restarting the Wacom driver I could for a short amount of time have access to the mouse while using FontForge.
So certainly a driver issue. I don't know what is behind the scene of FontForge technically speaking but if I could be pointed to what system is used to manage the UI I could report the issue to the relevant persons. You quoted GDK, that may be a good start, thanks.

**Comment by jtanx** [comment]
Can you please try the following? There's a way to enable (fairly verbose) debug logging of the GDK backend. Unfortunately the logging I wanted is commented out, but there are a couple of other log lines that may be interesting. As you say it's most likely something to do with the wacom driver, perhaps there's some events that we're not processing properly.

1. Open the FontForge interactive console
2. Enter:
   ```
   set GGDK_LOGLEVEL=all
   fontforge > C:\users\yourusername\Documents\log.txt
   ```
3. Try clicking around then close FontForge
4. Post log.txt back here

**Comment by jtanx** [comment]
Can you try one more thing:

1. Open the FontForge interactive console
2. Enter:
   ```
   set GDK_IGNORE_WINTAB=1
   fontforge
   ```
3. Check what works (mouse? tablet? both?)

**Comment by Requiem365** [comment]
Nice shot !
With the "set GDK_IGNORE_WINTAB=1" command all works fine so far. As well the tablet (no more menu issue) as the mouse.
This may be also related, like your link suggests, to the Windows Ink feature that I had to disable to use correctly Photoshop.
Thanks a lot !
Is there a way to launch the exe with that feature enabled by default or must I still use the console to avoid my issues (or will fontforge be updated) ?
And thanks again for the support.

Here is the log with the issue in case you still want to check (I had to copy paste since for some reason the log was empty despite running the console as administrator) :
[log.txt](https://github.com/fontforge/fontforge/files/6107534/log.txt)
